From 22dc3d0c49c9340866ce930b2f7861d2671cc965 Mon Sep 17 00:00:00 2001
From: Charalampos Stratakis <cstratak@redhat.com>
Date: Tue, 19 Jul 2016 14:38:14 +0200
Subject: [PATCH] Fix requests version bug

---
 linkcheck/__init__.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/linkcheck/__init__.py b/linkcheck/__init__.py
index 5624855..231a7c4 100644
--- a/linkcheck/__init__.py
+++ b/linkcheck/__init__.py
@@ -24,10 +24,10 @@ import sys
 # Needs Python >= 2.7.2 which fixed http://bugs.python.org/issue11467
 if not (hasattr(sys, 'version_info') or
         sys.version_info < (2, 7, 2, 'final', 0)):
-    raise SystemExit("This program requires Python 2.7.2 or later.")
+    import platform
+    version = platform.python_version()
+    raise SystemExit("This program requires Python 2.7.2 or later instead of %s." % version)
 import requests
-if requests.__version__ <= '2.2.0':
-    raise SystemExit("This program requires Python requests 2.2.0 or later.")
 
 import os
 # add the custom linkcheck_dns directory to sys.path
-- 
2.7.4

